po/ca.po

```diff
@@ -1582,7 +1582,7 @@ msgstr ""
 "cantó de les altres."
 
 #: pkg/packagekit/updates.jsx:951
-msgid "Cockpit itself will be updated."
+msgid "This web interface itself will be updated."
```

Comment:
Please don't update the *.po files. This breaks all existing translations, but that's just the price of it. We don't want to leave the translated messages "wrong" (in the sense of them still saying "Cockpit"). The next po-refresh bot task will take care of updating the template and updating the po file like that (but mark the strings as fuzzy).